pinot-core/src/main/java/org/apache/pinot/core/operator/docidsets/BitmapDocIdSet.java

```diff
@@ -27,6 +27,13 @@ public class BitmapDocIdSet implements BlockDocIdSet {
   private final BitmapDocIdIterator _iterator;
 
   public BitmapDocIdSet(ImmutableRoaringBitmap docIds, int numDocs) {
+    this(docIds, numDocs, false);
+  }
+
+  public BitmapDocIdSet(ImmutableRoaringBitmap docIds, int numDocs, boolean exclusive) {
```

Comment:
I'd suggest flipping the bitmap on the caller side before creating a `BitmapDocIdSet` which IMO is more clear

Reply:
Good suggestion. Done.